sandy/sections/mf2.py

```diff
@@ -0,0 +1,367 @@
+    out = {
+            "MAT": mat,         
+            "MF": mf,
+            "MT": mt,
+            }
+    i = 0
+    C, i = sandy.read_cont(df, i)
+    add = {
+            "ZA": C.C1,             
+            "AWR": C.C2,            #AWR is defines as the ratio of the mass of the material to that of the neutron
+            "NIS": C.N1,            #NIS is the number of isotopes in the material
+            }
+    out.update(add)
+    C, i = sandy.read_cont(df, i)
+    add = {
+            "ZAI": C.C1,
+            "ABN": C.C2,           #Abundance of an isotope in the material 
+            "LFW": C.L2,           #indication whether average fission wifths are given in the unresolbed resonance region
+            "NER": C.N1,           #Number of resonance energy ranges for this isotope. 
+            }    
+    LFW = int(C.L2) 
+    out.update(add)
+    
+    C, i = sandy.read_cont(df, i)
+    add = {
+            "EL": C.C1,             #Lower limit for an energy range
+            "EH": C.C2,             #Upper limit for an energy range
+            "LRU": C.L1,            #Flag indicating whether this energy range contains data for resolved or unresolved resonance parameters:
+            "LRF": C.L2,            #Flag indicating which representation has been used for the energy range. 
+            "NRO": C.N1,            #Flag designating possible energy dependence of the scattering radiu
+            "NAPS":C.N2,            #Flag controlling the use of the two radii
+            }
+    LRU = int(C.L1)
+    LRF = int(C.L2)
+    NRO = int(C.N1)
+    
+    out.update(add)
+    
+    print (LRU, LRF, NRO)
+    
+    if LRU == 0:
+        C, i = sandy.read_cont(df, i)
+        add = {
+                "SPI": C.C1,        #Spin, I, of the target nucleus.
+                "AP": C.C2,         #Scattering radius in units of 10e-12cm.
+                "NLS": C.N1,        #Number of l-values (neutron orbital angular momentum) in this energy region.
+                }
+        out.update(add)
+        
+    if LRU == 1:
+        if LRF == 1 or LRF == 2 :
+            
+            if NRO == 0 :
+                C, i = sandy.read_cont(df, i)
+                add = {
+                        "SPI": C.C1,
+                        "AP": C.C2,
+                        "NLS": C.N1,
+                        }
+                out.update(add)
+                L, i = sandy.read_list(df, i)
+                NRS= int(L.N2)
+                add = {
+                        "AWRI": L.C1,       #Ratio of the mass of a particular isotope to that of a neutron
+                        "QX": L.C2,         #Q-value to be added to the incident particle’s center-of-mass energy to determine the channel energy for use in the penetrability factor.
+                        "L": L.L1,          #Value of l.
+                        "LRX": L.L2,        #Flag indicating whether this energy range contains a competitive width
+                        "6*NRS": L.NPL,     
```

Comment:
Only "NRS" or another name. I dont remenber, but for `sandy.write_list` NPL or N2 was redundant.